tests/test_solution.py

```diff
@@ -79,6 +79,47 @@ def test_getSolTime():
     for s in m.getSols():
         assert m.getSolTime(s) >= 0
 
+def test_hasPrimalRay():
+    m = Model()
+    x = m.addVar()
+    m.setObjective(x, "maximize")
+    m.setPresolve(SCIP_PARAMSETTING.OFF)
+    
+    m.optimize()
+    
+    assert m.hasPrimalRay()
+
+    m = Model()
+    x = m.addVar(lb = 0) # for readability
```

Comment:
no need to add a comment here 😄 
```suggestion
    x = m.addVar(lb = 0)
```